_includes/master/charts/calico/values.yaml

```diff
@@ -0,0 +1,8 @@
+typha:
+  enabled: false
+network: "none"
+calico_ipam: false
+# datastore: "etcd"
```

Comment:
Do we need this file at all? Is it just to define defaults which we'll then overwrite?

Also, can we drop the commented lines?

Reply:
Yes it's to define defaults, and yes we can drop the commented lines.

But we still need the file and a few defined defaults as they represent the values the old jekyll system used to assume.